docs/self-managed/operational-guides/update-guide/870-to-880.md

```diff
@@ -6,6 +6,140 @@ description: "Review which adjustments must be made to migrate from Camunda 8.7.
 
 The following sections explain which adjustments must be made to migrate from Camunda 8.7.x to 8.8.x for each component.
 
+## Camunda Exporter
+
+The new Camunda Exporter brings the importing and archiving logic of web components (Tasklist and Operate) closer to the distributed platform (Zeebe). This simplifies the installation, enables scalability for the web applications, reduces latency when showing runtime and historical data, and reduces data duplication (resource consumption).
+
+![87-state](./img/87-state-refined.png)
+
+Camunda web applications use importers and archivers to consume, aggregate, and archive historical data provided by the Elasticsearch (ES) or OpenSearch (OS) exporters. The new Camunda Exporter introduces changes in the architecture users should be aware of.
+
+During the migration/update phase, the Operate and Tasklist importer and the Elasticsearch exporter are still required. This allows 8.7 data written to the Zeebe journal, or already in the Zeebe Elasticsearch indices, to be imported into the new harmonized index schema.
+
+![brownfield](./img/target-brown-field-refined.png)
+
+### Harmonized Indices
+
+The harmonized schema looks like the following:
+
+![harmonized-schema](./img/harmonized-indices-miro-v2.png)
+
+This reduces the need for several indices and to make use of existing ones (Operate indices are used by Tasklist, etc.)
+
+### Notes for updating to 8.8
+
+When updating from Camunda 8.7 to 8.8, the following should be considered regarding the Camunda Exporter and harmonized indices.
+
+#### Ensure the exporter/import backlog is small
+
+For data from 8.7 to be imported, the importers must run first and complete before the Camunda Exporter can start.
+
+The importer reads data from existing Zeebe Elasticsearch indices, and when it detects data from newer versions, it marks itself as complete. The Camunda Exporter then starts exporting only when the importer is done.
+
+:::important
+To reduce the potential of building a backlog,
+the user needs to make sure to drain the backlog queue as much as possible before the update.
+:::
```

Comment:
Not necessary here, but since the header for this section is on backlog size, it would be nice to know what would happen if the backlog is too large and you attempt to upgrade anyway. 

```suggestion
For data from 8.7 to be imported, the importers must first run and complete before the Camunda Exporter can start. The importer reads data from existing Zeebe Elasticsearch indices, and when it detects data from newer versions, it marks itself as complete. The Camunda Exporter then starts exporting only when the importer is done.

If the import backlog is too large, [why do we require the backlog to be small].

:::important
To reduce the potential of building a backlog, drain the backlog queue as much as possible before the update.
:::
```